packages/PHPStanStaticTypeMapper/TypeMapper/TypeWithClassNameTypeMapper.php

```diff
@@ -39,6 +45,10 @@ public function mapToPHPStanPhpDocTypeNode(Type $type, ?string $kind = null): Ty
      */
     public function mapToPhpParserNode(Type $type, ?string $kind = null): ?Node
     {
-        return $this->stringTypeMapper->mapToPhpParserNode($type, $kind);
```

Comment:
I had to inline the impl of `stringTypeMapper->mapToPhpParserNode()` because with the added interface the StringTypeMapper no longer access the given `TypeWithClassName` arg